### PR TITLE
Create an endpoint to query all roles.

### LINKED ISSUE #2983: Create an endpoint to query all roles
* Add `GetAllListAsync()` method to the `IIdentityRoleAppService`
* Set route as `[Route("all")]` in the `IdentityRoleController`
* Use this method from MVC UI to query roles when assigning to a user

Based on https://github.com/abpframework/abp/issues/2629

### PR DESCRIPTION
**Pull request by maliming**
Resolve #2983

### CODE REVIEW

File: modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IIdentityRoleAppService.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ namespace Volo.Abp.Identity
 {
     public interface IIdentityRoleAppService : IApplicationService
     {
+        Task<List<IdentityRoleDto>> GetAllListAsync();
```
Comment (hikalkan) [inline modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IIdentityRoleAppService.cs]: Please use ListResultDto wtihh has Items collection. We use this "wrapping" to be able to add new properties to the object without breaking existing client applications (backward compability).

Comment (maliming) [reply modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IIdentityRoleAppService.cs]: `ListResultDto` has been used.

**Comment by maliming** [comment]
@hikalkan Already done.

**Comment by hikalkan** [comment]
Thanks!

**Pull request merged**